data/nlu/nlu.md

```diff
@@ -4931,25 +4975,24 @@
 ## synonym:stack
 - fullstack
 - Full Stack
-- full stack
-- full
 - Full
+- full stack
+- full staclk
```

Comment:
Did you edit/test via a local rasa x or a deployed one? if the former, let's remove any purely formatting-based stuff that was changed

Reply:
I did not use a local rasa x, so I will leave these as is. I wonder if deployed rasa x also does these type of formatting-based changes.

Reply:
The issue is that they format stuff differently because I think the formatting PR you approved today fixed the discarding issue. 

I think it was introduced when greg must have saved from local here: https://github.com/RasaHQ/rasa-demo/pull/420/files

and then rasa x kept saying there was a diff and trying to change it back. Should create a PR once i've collected the differences in what it allows, because that's quite annoying to the dev workflow

Reply:
feel free to resolve, wanted you to see the new comment